mergin/client_pull.py

```diff
@@ -183,6 +185,8 @@ def download_project_is_running(job):
     """
     for future in job.futures:
         if future.done() and future.exception() is not None:
+            job.mp.log.error("Error while downloading project: " + str(future.exception()))
```

Comment:
It would be good to produce more details here - including traceback, because it could be arbitrary exception...